test/include/test_runner/test_case.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "main/kuzu.h"
+
+using namespace kuzu::main;
```

Comment:
Remove `using namespace kuzu::main;`. Our convention is that we never use `using namespace` in header files, but you can use them in cpp files. In header files, we add the namespace prefix as needed.

Reply:
Should I do the same with `test_helper.h` as well?

Reply:
Up to you. Just keep a note is fine.